test/e2e/flowaggregator_test.go

```diff
@@ -427,6 +429,18 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
 		}
 	})
 
+	// ToExternalFlows tests the case, where Pod is deployed on the source Node and
+	// sending traffic to an external IP, the flow information is exported as IPFIX
+	// flow records.
```

Comment:
not really a correct sentence, did you just mean:

> ToExternalFlows tests the export of IPFIX flow records when a source Pod sends traffic to an external IP

Reply:
I was following the same format of other test descriptions in the file.
Should I rephrase other descriptions as well? also noticed the other inconsistency among the comments, e.g. some with trailing periods, some without, some start with caps, some not.